governance.md

```diff
@@ -1,15 +1,103 @@
+
+### Removing a steering committee member
+
+If a steering committee member is no longer interested or cannot perform the duties listed above, they should volunteer to be moved to emeritus status. Once volunteered, steering committee will call for a [vote](#Voting). In extreme cases, removing existing member can also occur by a vote of the steering committee.
+
+### Nonaffiliated individual
+
+Individuals not associated with or employed by a company or organization are allowed as members of steering committee while this is considered as **rare case**. Clarification will be required in the nomination PR to explain its necessity and must be verified and approved by steering committee.
+
+## Maintainer Role
+
+Maintainer role is per project. Maintainers have the most experience with the project under `oam-dev` organization and are expected to lead its growth and improvement.
+
+Responsibilities include:
+
+* Strong commitment to the project.
+* Participate in design and technical discussions.
+* Contribute non-trivial pull requests.
+* Perform code reviews on other's pull requests.
+* Regularly triage GitHub issues.
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.
+* Monitor OAM mailing list, slack channel and response to social media interactions.
+* Regularly attend the community meetings.
+
+The current list of maintainers is published and updated in `OWNERS.md` of every project.
+
+### Becoming a maintainer
+
+To become a maintainer of [OAM specification](https://github.com/oam-dev/spec): 
+- OAM specification is maintained by the OAM steering committee members **only**. Please check the [requirements of becoming steering committee member](#Becoming-a-Steering-Committee-Member).
+
+
+To become a maintainer of other projects under `oam-dev` organization:
+
+- Active contributions for at least 1 month.
+- Get at least 10 non-trivial pull requests merged to the codebase.
+- Primary reviewer for at least 3 non-trivial pull requests from others.
+- Done through PR to update the project’s OWNERS file.
+- Self-nominate or be nominated by existing maintainers in the project.
+- With no objections from existing maintainers in the project.
+
+### Removing a maintainer
+
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status. Once volunteered, it will automatically take effect unless objection is raised from existing maintainers. In extreme cases removing maintainers can also occur by a vote of steering committee.
+
+
+### Github Project Administration
+
+Maintainers will be added to the OAM GitHub organization (if they are not already) and added to the GitHub Maintainers team.
```

Comment:
Is the maintainers team at the organization level? it seems like team usage here may not work well since each project can/will have different maintainers, so having 1 maintainers team at the org level wouldn't be granular enough.  Perhaps adding members directly to each repo's `/settings/access` page may work better? Or having a team per project? 

What level of permission do they get, "maintain"? (i think that's a fairly new rule that github allow now)

Reply:
Maintainers are per project, so I believe "a maintainer team per project" is the practice. I will elaborate this in this section.